node/network/availability-distribution/src/requester/fetch_task/mod.rs

```diff
@@ -281,6 +287,11 @@ impl RunningTask {
 					continue
 				},
 			};
+			drop(_chunk_fetch_span);
```

Comment:
Why do you need to drop manually? Not just here but wondering in general.

Reply:
By default spans are dropped when they go out of scope. However, there are sometimes where we can manually drop spans. It is a bit of an art since it is based on how long the observer wants a particular span to be alive. In this case, it is done directly because the `_chunk_fetch_span` only captures the duration of the request sent to get a chunk, and not anything else.

Reply:
@bredamatt Documenting that in a comment seems like it would be good! In general, manual `drop`s should be accompanied with some justification IMO.

Reply:
@alexgparity yes, you are right at a high-level. This is somewhat how it works under the hood, but obviously with more detail as it is based on which tracing stack is used. Usually this is referred to as sampling and focuses on the performance tuning: https://www.jaegertracing.io/docs/1.23/performance-tuning/

In our case, there is an external crate which handles how spans are being passed to Grafana Tempo - this is the `mick-jaeger` crate, a wrapper around `jaeger`. Changing `mick-jaeger` is not really in scope for this PR since we are using `mick-jaeger` throughout all subsystems from beforehand.